### DISCUSSION TITLE
farming rewards

### DISCUSSION THREAD

**Discussion by bastetbar** [Chia-Network/chia-blockchain#17696] [Support]
After the update, the farming rewards were halved. There were no changes in my harvester. Why did it decrease from 1.3 to 0.6? Difficulty same in the Spacepool.

**Comment by BrandtH22** [comment]
Hey @bastetbar ,

The halving has recently occurred so it is expected to be receiving 1/2 of the previous rewards. 

More information about the halving can be found here:
https://www.chia.net/2024/02/15/chia-blockchain-explainer-the-halvings/
